students/Ben_freeman/lesson06/mailroom4_test.py

```diff
@@ -0,0 +1,35 @@
+import mailroom4
+import os
+dummy_dictionary={"Test1":[1,2,3]}
+def test_1():
+    assert mailroom4.send_thanks_print("Santa",20)==print("\nThank you Santa for your donation of $20, your donations make the work of the 'American society for taking donations' possible.\n\n"
+                                                "Sincearly,\n\n"
+                                                "A low paid intern\n")
+
+def test_2():
```

Comment:
ideally your tests should have a human readable name, these tests are usually aggregated into reports.


for example -> `def test_send_thanks_list`